pom.xml

```diff
@@ -27,10 +27,11 @@
         <commonsIoVersion>2.8.0</commonsIoVersion>
         <mockito-core.version>3.9.0</mockito-core.version>
         <xmlunit.version>1.6</xmlunit.version>
-        <caffeineVersion>2.9.0</caffeineVersion>
```

Comment:
Please let's review this next week: can't find a way to solve the issue when taking out duplicate dependencies from the connector (it compiles well) since the test-suit (DbArtifactClassLoaderRunnerConfig) needs them.

Reply:
I took a look at the errors and the JMS Connector for reference. 
First of all.. this seems to be a sharedRuntimeLib issue. 
`-------------------------------------------------------------------------------
Test set: org.mule.extension.db.integration.function.oracle.CreateArrayOracleTestCase
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0 s <<< FAILURE! - in org.mule.extension.db.integration.function.oracle.CreateArrayOracleTestCase
initializationError(org.mule.extension.db.integration.function.oracle.CreateArrayOracleTestCase)  Time elapsed: 0 s  <<< ERROR!
java.lang.IllegalStateException: Application shared lib artifact 'com.mchange:c3p0' in order to be resolved has to be declared as test dependency of your Maven project (org.mule.connectors:mule-db-connector:mule-extension:2.0.0-SNAPSHOT)

`
Then looking at the JMS Connector and the JMS Client this seems to be a dependency of both of them and is exported as a sharedRuntimeLibrary by the JmsAbstractTestCase.
We should leave this as it is. Please dismiss this change request.

Reply:
ok, yes I checked jms and see duplicated dependencies.
Later we should have a discussion on how to use ArtifactClassLoaderRunnerConfig and how dependencies affect this.

thanks!